css/src/atomics/README.md

```diff
@@ -0,0 +1,54 @@
+# Atlas Atomics
+
+Welcome to Atomics ⚛! These are small, single purpose, css classes that can be used to compose great things. Visit this article for a [longer definition of Atomic CSS here](https://css-tricks.com/lets-define-exactly-atomic-css/#:~:text=Atomic%20CSS%20is%20the%20approach,names%20based%20on%20visual%20function).
+
+## Categories of classes
+
+Most of our classes fall into one of several categories. Those that affect ...
+
+- display (ex. `.display-inline-flex`)
+- spacing (ex. `.margin-inline-l`);
+- color (ex. `.color-primary`)
+- typography (ex. `.font-size-m`);
+- flex properties (ex. `.justify-content-center`)
+- and more ...
+
+## Naming patterns
+
+All of our atomic classes try to follow the same pattern.
+
+```txt
+.<css-property-name>-<value>
+// or
+.<css-property-name>-<value>-<screensize>
+```
+
+## CSS Property Values
+
+The first part of nearly every Atomic class represent the css property that it will modify. For example, `display` in the class `.display-flex` matches the property value from the corresponding css rule `display: flex;`. We hope this will encourage everyone to provide a clear and easily intuited pattern for our Atmoics—and encourage everyone to learn the fundamentals of CSS.
```

Comment:
Atmoics -> Atomics

"We hope this will provide a clear and easily intuited pattern for our Atomics and encourage everyone to learn the fundamentals of CSS." ?

Reply:
Should `css` always be `CSS`?

Reply:
updated, and yes to `CSS`